README.md

```diff
@@ -114,6 +115,15 @@ for more testing insights.
 - Explore and utilise [Docker healthcheck][docker-healthcheck]
 - Reduce setup steps - can Specialist Publisher work without finders for instance?
 
+## Gotchas
+
+### WEBrick server seems to stop responding
+
+We had a nasty bug with Router API where the web server seemed to stop
+responding to any requests. This issue was experienced once the application
+was upgraded to Rails 5.1 and Mongoid 6.1. The symptom was that any requests
+to the server seemed to hang and never respond. The way this was resolved was
+to switch from using WEBrick on Router API and instead use unicorn server.
```

Comment:
Perhaps link to the PR which introduced this fix to help for the future?

Reply:
Good idea